pkgs/by-name/ch/chatd/package.nix

```diff
@@ -0,0 +1,88 @@
+{ buildNpmPackage
```

Comment:
Use `nixfmt` (rfc)

Reply:
nixpkgs-fmt on the file is doing a no-op, so it seems it's right

Reply:
found out how to do that, now it's fixed